docs/source/conf.py

```diff
@@ -46,9 +46,8 @@
 
 # The default replacements for |version| and |release|, also used in various
 # other places throughout the built documents.
-base_path = os.path.dirname(__file__)
-version_file = os.path.join(base_path, '..', '..', 'enable', '_version.py')
-version = release = runpy.run_path(version_file)['full_version']
+from enable import __version__
+version = release = __version__
```

Comment:
I vaguely remembered there is a reason why it was done via executing the code...

The one difference I know (not sure if it is the same as the reason I think I forgot): On master, you did not have enable installed, then building documentation would still "work", but the resulting documentation does not contain any API documentation because it relies on autodoc. Here you must have enable installed to build anything at all.

Reply:
I tested that without cleaning my local git repository. If I clean the local git repository, then `_version.py` would have been removed, but building the documentation would also fail with the original code.
But the source distribution includes `_version.py`, so it would work there without having to install enable source first. It might have been for that.

Reply:
If `_version.py` is not available as suggested in the PR description, I am puzzled by how the new change is supposed to work.